sharpy/io/logger_utils.py

```diff
@@ -0,0 +1,40 @@
+import logging
```

Comment:
The logging module could be very interesting to implement for the future. 

For now, it is very useful in the udp module to see when things are received and sent. As for the debug messages, in these case they are "user debug" rather than "developer debug". The use of the debug level in the udp module is intended for the user to check connections between computers as well as signals being sent or received. Once happy it can be turned off to keep the screen verbose to a minimum

Reply:
I see, I am sure it is a source of problems and this is helpful. Thanks